src/NzbDrone.Core.Test/IndexerTests/NewznabTests/NewznabRequestGeneratorFixture.cs

```diff
@@ -339,5 +339,39 @@ public void should_fallback_to_q()
             pageTier2.Url.Query.Should().NotContain("rid=10");
             pageTier2.Url.Query.Should().Contain("q=");
         }
+
+        [Test]
+        public void should_encode_raw_title()
+        {
+            _capabilities.SupportedTvSearchParameters = new[] { "q" };
+            _capabilities.TextSearchEngine = "raw";
+            _singleEpisodeSearchCriteria.SceneTitles[0] = "Edith & Little";
+            var results = Subject.GetSearchRequests(_singleEpisodeSearchCriteria);
+            results.Tiers.Should().Be(1);
+
+            var pageTier = results.GetTier(0).First().First();
+
+            pageTier.Url.Query.Should().Contain("q=Edith%20%26%20Little");
+            pageTier.Url.Query.Should().NotContain(" & ");
+            pageTier.Url.Query.Should().Contain("%26");
+        }
+
+        [Test]
+        public void should_use_clean_title_and_encode()
+        {
+            _capabilities.SupportedTvSearchParameters = new[] { "q" };
```

Comment:
Same here: This test is failing because it's missing `season` and `ep` parameters.